internal/kafka/command_topic_produce.go

```diff
@@ -632,3 +645,21 @@ func setSchemaPathRef(schemaString srsdk.SchemaString, dir, subject string, sche
 	}
 	return tempStorePath, referencePathMap, nil
 }
+
+func parseHeaders(headers []string, delimiter string) ([]ckafka.Header, error) {
+	var kafkaHeaders []ckafka.Header
+
+	for _, header := range headers {
+		parts := strings.SplitN(header, delimiter, 2)
+
+		key := strings.TrimSpace(parts[0])
+		if key == "" {
+			return nil, fmt.Errorf(invalidHeadersErrorMsg, delimiter)
+		}
+
+		if len(parts) == 2 {
```

Comment:
It looks like we currently ignore an element in the list if it doesn't have a delimiter. Can we return an error if it's missing a ":"?

Reply:
Yeah you are correct. I have handled this case. Thanks for pointing it out.